depthai_sdk/src/depthai_sdk/__init__.py

```diff
@@ -0,0 +1,4 @@
+from .fps import *
+from .previews import *
+from .utils import *
+from .managers import *
```

Comment:
Does this remove the 'namespaces' when doing `import depthai_sdk` so `depthai_sdk.managers.PipelineManager` is then `depthai_sdk.PipelineManager`?

Just sane checking if this is a good idea

Reply:
It's just for convenience - so if someone wants to have everything imported in one line, he still can
```python
from depthai_sdk import to_tensor_result, Previews, PipelineManager
```

Personally, I like to split the declarations into managers/others, like this

```python
from depthai_sdk.managers import PipelineManager
from depthai_sdk import to_tensor_result, Previews
```

However, someone could also be very strict with his imports and do like this

```python
from depthai_sdk.managers.pipeline_manager import PipelineManager
from depthai_sdk.utils import to_tensor_result
from depthai_sdk.previews import Previews
```

And actually, the code snippet above shows how the package would be imported without the imports in `__init__.py`, which allows previous snippets to work.
So it's more for users convenience and allowing them to import the package as preferred

Reply:
But isn't `__init__.py` always called when `depthai_sdk` import is called?

Eg:
```
from depthai_sdk.managers import PipelineManager
from depthai_sdk import to_tensor_result, Previews
```

The above would include Previews into `depthai_sdk` right?

Anyway, as long as there aren't name clashes between `managers`, `previews`, `utils`, etc... its probably okay

Reply:
Yes, but the following import
```
from depthai_sdk.managers import PipelineManager
```
Would not work if `managers/__init__.py` would not contain the proper import, as this class is in `mamagers/pipeline_manager.py` file